subprojects/docs/src/docs/release/notes.md

```diff
@@ -67,7 +67,21 @@ For example, in the following crash, the JVM fatal error log was written into `/
 
 ## Default location for native headers generated from Java source
 
-The Java compile task for each Java source set now has a default location to generate native headers into. This is used by the Java compiler when the Java source contains `native` methods.  
+The Java `native` keyword allows to declare methods that are implemented in native code.
+The Java compiler can then generate native headers that should be implemented.
+This mechanism supports using the [Java Native Interface (JNI)](https://docs.oracle.com/en/java/javase/11/docs/specs/jni/index.html) specification but is not limited to it.
+
+The Java compile task for each Java source set now has a default location to generate native headers into.
+
+This location can also now be changed if need be: 
```

Comment:
Why a user might want to do it?

Reply:
Because some other tool would expect them in some given location.
But we can leave that out, the DSL reference explains you can set it if need be.

Reply:
Let's say that. That's what release notes are for. 

For example:
"Some tools may expect header files in a different location. It can be changed now if needed:"

Reply:
Done